fuzzbench/jobs.py

```diff
@@ -11,25 +11,30 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-"""Fake jobs."""
+"""Jobs."""
 
+import os
 import subprocess
-import time
 
+BASE_TAG = 'gcr.io/fuzzbench'
 
-def build_image(name):
-    """Build a Docker image."""
-    print('Building', name)
-    subprocess.run(['docker', '--version'], check=True)
-    time.sleep(3)
+
+def build_image(name: str):
+    """Builds a Docker image."""
+    image_tag = os.path.join(BASE_TAG, name)
```

Comment:
This function is not going to be sufficient for building every image. Is that fine?

Reply:
>This function is not going to be sufficient for building every image.

No, it is not sufficient.

>Is that fine?

Currently, it should be fine. We may add more logic when we need.

Reply:
(for more context: as mentioned before once https://github.com/google/fuzzbench/pull/543 lands we can use that here without re-implementing the build logic)